services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -64,7 +62,7 @@ async def _abort_pipeline_tasks(
     computation_tasks: CompTasksRepository,
```

Comment:
so this file should be "celery or dask agnostic". All references should be abstracted to the base scheduler, right?

- should not have any concrete imports e.g. ``from ...modules.celery import CeleryClient``
- ``_abort_pipeline_tasks`` is NOT used anymore

Reply:
very good ones. I forgot those.